src/hyperion/experiment_plans/panda_flyscan_xray_centre_plan.py

```diff
@@ -189,9 +189,11 @@ def run_gridscan_and_move(
         time_between_x_steps_ms,
     )
 
-    get_directory_provider().update(
-        directory=Path(parameters.detector_params.directory)
-    )
+    panda_directory = Path(parameters.detector_params.directory)
```

Comment:
Should: I think `parameters.storage_directory` is where we get the detector directory from originally so we should probably use that here rather than digging into the detector.

Reply:
Should: I'm also mildly concerned that people will get detector and panda data confused. Can we put it in a `panda` subfolder for now?